extensions/data-plane-transfer/data-plane-transfer-sync/src/test/java/org/eclipse/dataspaceconnector/transfer/dataplane/sync/validation/ContractValidationRuleTest.java

```diff
@@ -44,8 +45,15 @@ class ContractValidationRuleTest {
     private final Instant now = Instant.now();
     private final Clock clock = Clock.fixed(now, UTC);
 
-    private final ContractNegotiationStore contractNegotiationStore = mock(ContractNegotiationStore.class);
-    private final ContractValidationRule rule = new ContractValidationRule(contractNegotiationStore, clock);
+    private ContractNegotiationStore contractNegotiationStore;
+    private ContractValidationRule rule;
+
+
+    @BeforeEach
+    public void setUp() {
+        contractNegotiationStore = mock(ContractNegotiationStore.class);
```

Comment:
the mock has no reason to be instantiated in the setup, otherwise also the same should be done also for the clock, but this will add unnecessary noise, when the most important part here is the test, as the setup should be straightforward.